typehints/esp8266/dht.pyi

```diff
@@ -0,0 +1,61 @@
+"""DHT driver
+
+Driver enables temperature and humidity readings from the DHT-11 and DHT-22 (also know as AM2302).  Humidity is
+measured as percentage of relative humidity and temperature is measured in Celsius.
+
+"""
+
+
+class DHTBase:
+    def __init__(self, pin: object) -> None:
+        """
+        Initialise base class for both DHT-11 and DHT-22
+        :param pin: GPIO pin object providing 1-wire data signal
+        """
+
+    def measure(self) -> None:
+        """
+        Updates temperature and humidity measurements from the sensor and stores them them ready for reading.
+
+        To ensure accurate results:
+        * DHT-11 should have at least a 1 second wait between calls
+        * DHT-22 should have at least a 2 second wait between calls
+        """
+
+
+class DHT11(DHTBase):
+    def humidity(self) -> bytearray:
```

Comment:
I believe the return type of this method is `int` (as well as the return type of other `humidity()` and `temperature()` methods. Could you please check it with `type(DHT11(...).humidity())`?

Reply:
I can't check the DHT-11 as I only have the DHT-22 sensor however, given the return type on the HT-22 is float and not bytearray, it seems reasonable that the DHT-11 will return int.  I'll make the changes.